explainer.md

```diff
@@ -139,20 +176,66 @@ let depthValueInMeters = depthInfo.getDepth(x, y);
 
 ## Appendix: Proposed Web IDL
 
+The comments are left in place to provide additional context on the API to the readers of the IDL and should also be repeated in the explainer text above.
+
 ```webidl
+enum XRDepthSensingUsage {
+  "cpu-optimized",
+  "gpu-optimized"
+};
+
+enum XRDepthFormat {
+  // Has to be supported everywhere:
+  "luminance-alpha", // internal_format = LUMINANCE_ALPHA, type = UNSIGNED_BYTE,
+                     // 2 bytes per pixel, unpack via dot([R, A], [255.0, 256*255.0])
+  // Other formats do not have to be supported by the user agents:
+  "float32",         // internal_format = R32F, type = FLOAT
+};
+
+// Post-session-creation, the depth sensing state needs to be configured
+// with the desired data format.
+dictionary XRDepthSensingStateInit {
+  XRDepthSensingUsage usage = "cpu-optimized";
+  XRDepthFormat depthFormat = "luminance-alpha";
+};
+
+partial interface XRSession {
+  // Non-null iff depth-sensing is enabled:
+  readonly attribute XRDepthFormat? preferredDepthFormat;
+  readonly attribute XRDepthSensingUsage? preferredDepthSensingUsage;
+
+  // Can be called iff depth-sensing is enabled, will throw if unsupported format is passed in.
```

Comment:
`iff`

Reply:
[By design](https://en.wikipedia.org/wiki/Iff), "iff" = "if and only if".